File: api/models/Regiao.php
<?php

namespace Model;

class Regiao extends \Database\Mysql {
    
    public $estados = [];
    public $estados_siglas = [];
    public $municipios = [];

    function __construct() {
        parent::__construct();
    }

    function get_estados_siglas() {
        $estados = $this->fetch_array("SELECT sigla FROM estados");
        $siglas = [];
        foreach($estados as $estado) array_push($siglas, $estado['sigla']);
        $this->estados_siglas = $siglas;
    }
}

File: api/models/regiao.py
from mysql import connection
import json


class Regiao:

    def json(self, data):
        return json.dumps(data, indent=4)

    def get_estados_siglas(self):
        with connection.cursor() as cursor:
            siglas = []
            cursor.execute("SELECT sigla FROM estados")
            res = cursor.fetchall()
            for item in res:
                siglas.append(item['sigla'])
            return self.json(siglas)

    def get_municipios_by_uf(self, uf):
        with connection.cursor() as cursor:
            items = []
            cursor.execute("SELECT * FROM comarcas WHERE uf = '{uf}'".format(uf=uf))
            res = cursor.fetchall()
            for item in res:
                items.append({"codibge": item['codibge'], "nome": item['nome']})
            return self.json(items)

